salsa_dancing_molecules/lattice_constant_and_bulk_modulus.py

```diff
@@ -0,0 +1,43 @@
+"""
+Function for calculating lattice constant and bulk modulus.
+
+Takes a trajectory file and outputs lattice constant (in Angstrom)
+and bulk modulus (in GPa) as float values.
+"""
+
+from ase.io import read
+from ase.units import kJ
+from ase.eos import EquationOfState
+
+
+def get_lattice_constant_and_bulk_modulus(traj_file):
+    """
+    Read traj file, place atoms objects in list and call calculation function.
+
+    Takes a filepath to traj file as a string and returns lattice constant and
+    bulk modulus.
+    """
+    configs = read(traj_file)
+    calculate_lattice_constant_and_bulk_modulus(configs)
+
+
+def calculate_lattice_constant_and_bulk_modulus(configs):
+    """
+    Calculate lattice constant as done in ASE documentation.
+
+    Takes a list of atoms objects at different time steps and returns
+    a - lattice constant, and B - bulk modulus.
+    """
+    volumes = [atom.get_volume() for atom in configs]
+    energies = [atom.get_potential_energy() for atom in configs]
+    eos = EquationOfState(volumes, energies)
+    v0, e0, B0 = eos.fit()
+    a = (4*v0) ** (1/3)
```

Comment:
This calculations is only true for FCC, we need to be able to handle BCC/SC aswel. Maybe use a if-statement and check what structure the atom object is?